docs/resources/resource_group.md

```diff
@@ -0,0 +1,28 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "oktapam_resource_group Resource - terraform-provider-oktapam"
+subcategory: ""
+description: |-
+  
+---
+
+# oktapam_resource_group (Resource)
+
+
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `delegated_resource_admin_groups` (List of String) The ids of the groups for users who are designated as delegated admins for the resource group
```

Comment:
We don't call these objects delegated resource admin groups in the UI. They are called owners in the Resource Groups UI. I think we should use the same name for this construct here otherwise customers have to map between the concept of a resource group owner and this field name.

Reply:
I get where Walter is coming from, and I agree if the API, backend, database columns, etc  also calls them owners. Otherwise I'd follow what the API calls them as the UI/UX can change, but we'll likely not seriously invest into renaming the API calls.

Reply:
Based on conversations offline, we're going to keep the use of 'delegated resource admins' here and change the UX to match accordingly.